src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,5 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+export const UserInfo = ({ user: {email, name} }) => (
```

Comment:
Make sure that you are handling a case when the user prop is undefined. If the user object is undefined, destructuring will cause an error. You can either add a default value for the user prop when you're destructuring or handle it in the component logic.